src/components/controls/number/NumberCSS.js

```diff
@@ -0,0 +1,25 @@
+import { colors } from "../../colors";
+
+const ContainerCSS = `
+  display: flex;
+  border-radius: 50%;
+  cursor: pointer;
+  border: 2px solid ${colors.midnight400};
+  height: 34px;
+  width: 34px;
+  justify-content: center;
+  align-items: center;
```

Comment:
I'd add `font-family: "HK Grotesk", sans-serif` here to follow the design. Otherwise, looks good

Reply:
fixed